security/rbac/delta_configs/server.properties.delta

```diff
@@ -43,3 +43,8 @@ listener.name.rbac.oauthbearer.sasl.jaas.config= \
 listener.name.rbac.oauthbearer.sasl.server.callback.handler.class=io.confluent.kafka.server.plugins.auth.token.TokenBearerValidatorCallbackHandler
 # Set SASL callback handler for handling tokens on login. This is essentially a noop if not used for inter-broker communication.
 listener.name.rbac.oauthbearer.sasl.login.callback.handler.class=io.confluent.kafka.server.plugins.auth.token.TokenBearerServerLoginCallbackHandler
+
+# Setup MetricsReporter (point at PLAINTEXT port)
```

Comment:
@brianbushree why is MetricsReporter pointing at the PLAINTEXT port (9093) instead of the RBAC port (9092)